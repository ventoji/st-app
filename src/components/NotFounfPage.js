import React from 'react';
import { Link } from 'react-router-dom';
import Layout from '../components/Layout';

const NotFoundPage = () => (
  <Layout>
    404 - <Link to="/"> Go Home </Link>
  </Layout>
);

export default NotFoundPage;
