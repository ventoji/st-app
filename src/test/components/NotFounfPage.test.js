import React from 'react';
import NotFounfPage from '../../components/NotFounfPage';
import { toMatchSnapshotFunction } from '../fixture/helper';

test('should render Header coorectly', () => {
  toMatchSnapshotFunction(<NotFounfPage />);
});
