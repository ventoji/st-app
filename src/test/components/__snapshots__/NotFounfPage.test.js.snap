// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header coorectly 1`] = `
<Layout>
  404 - 
  <Link
    to="/"
  >
     Go Home 
  </Link>
</Layout>
`;
